packages/twenty-front/src/modules/navigation/components/MainNavigationDrawerItems.tsx

```diff
@@ -20,6 +20,11 @@ const StyledMainSection = styled(NavigationDrawerSection)`
   min-height: fit-content;
 `;
 
+const StyledContainer = styled.div`
+  overflow-x: hidden;
```

Comment:
why? This looks not OK! (we should use ScrollWrapper to scroll, but I think @lucasbordeau has already fixed it)

Reply:
not needed anymore I thik

Reply:
This needs a review plus help @lucasbordeau!
In this case, I couldn't figure out how to add the scroll wrapper.
This is bugged in this PR --- needs to be resolved before merge!

Reply:
Ah! I got it! nvm! :)

Reply:
The dragging behaviour gets messed up when the wrapper is added!